projects/storefrontlib/src/lib/cms-lib/product-carousel/product-carousel.component.spec.ts

```diff
@@ -18,7 +18,7 @@ describe('ProductCarouselComponent', () => {
   let fixture: ComponentFixture<ProductCarouselComponent>;
   let el: DebugElement;
 
-  const productCodeArray = ['111111', '222222', '333333', '444444'];
+  const productCodeArray: string[] = ['111111', '222222', '333333', '444444'];
   const mockComponentData = {
```

Comment:
Can we add a type?

Reply:
Can you add a type to line 34?

Reply:
Can you convert `MockCmsService` and `MockProductService` to classes? This would require to change `useValue` to `useClass` in the testing module config.